ci-operator/step-registry/hypershift/aws/e2e/cluster/OWNERS

```diff
@@ -0,0 +1,13 @@
+approvers:
```

Comment:
This file is auto generated ?  do we need to align it ?

Reply:
No it's not. I copy-paste the original owners since I copied this file verbatim, but a single line I deleted